try :
    nu = int(input("Enter your number : "))
    print(num)

except NameError as ex:
    print("Exception : ", ex)

    print("I am outside the block")
